### ISSUE TITLE
R configuration?

### ISSUE THREAD

**Issue by cgmurray** [lassefolkersen/my-statistical-health#1]
It looks like this was built on Ubuntu. Could you provide some instructions for installing and configuring R for your app, and perhaps some sample data? Did you just use a standard CRAN install?

**Comment by lassefolkersen** [comment]
Yes, I can. But I actually just meant this github-code for algorithm documentation and reference. It is running off a Amazon EC2 (yes Ubuntu, as you noted), at 
www.my-statistical-health.com so no need to run your own instance
